projects/core/src/product/store/selectors/product.selectors.ts

```diff
@@ -15,36 +15,21 @@ export const getProductState: MemoizedSelector<
   EntityLoaderState<Product>
 > = createSelector(getProductsState, (state: ProductsState) => state.details);
 
-export const getSelectedProductsFactory = (
```

Comment:
Info in migration guide about removing that?

Reply:
👍